src/features/decoration/di.config.ts

```diff
@@ -0,0 +1,30 @@
+/********************************************************************************
+ * Copyright (c) 2017-2018 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { configureModelElement } from "../../base/views/view";
+import { ContainerModule } from "inversify";
+import { SIssueMarker } from "./model";
+import { IssueMarkerView } from "./views";
+import { TYPES } from "../../base/types";
+import { DecorationPlacer } from "./decoration-placer";
+
+const decorationModule = new ContainerModule(bind => {
+    configureModelElement({bind}, 'marker', SIssueMarker, IssueMarkerView);
```

Comment:
Better choose a more specific type string such as `issue-marker`?

Reply:
I like marker better (coming from Eclipse). Note that we usually allow a `:` to separate base type and sub type. I'd find introducing a `-` a bit confusing.